.gitignore

```diff
@@ -21,4 +21,20 @@ __pycache__/
 
 #idea/pycharm
 .idea/
-.tox
\ No newline at end of file
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.pytest_cache/
+
+# testapp's build folder
+testapps/build/
+
+# Dolphin (the KDE file manager autogenerates the file `.directory`)
+.directory
```

Comment:
are you trying to sneak your file manager leftover to this PR :trollface:

Reply:
You caught me :laughing:, anyway...probably will be useful for someone else don't you think :wink:?